test-framework/test_framework_test.go

```diff
@@ -393,24 +393,6 @@ func TestUsingEnv(t *testing.T) {
 		require.NoError(t, result.Error)
 	})
 
-	t.Run("auth account", func(t *testing.T) {
```

Comment:
Why was the test removed?

Reply:
`getAuthAccount` function was available before, because the base activation had it all. But now the base activation doesn't have it. Could use `NewScriptInterpreterEnvironment`, but it returns `Environment` interface, which means checker/interpreter configs cannot be customized.
On the other hand, tests run locally and not on a chain, so they don't really have any accounts to access.

Reply:
Makes sense. Agreed 👍